app/controllers/api/v1/offers_controller.rb

```diff
@@ -80,7 +80,10 @@ def search
 
       api :GET, '/v1/offers/1', "List an offer"
       def show
-        render json: offer_serializer.new(@offer, exclude_messages: params["exclude_messages"] == "true").as_json
+        render json: offer_serializer.new(@offer, {
+          include_organisations_users: params["include_organisations_users"] == "true",
```

Comment:
@abulsayyad123 cc @patrixr Patrick has a fix coming for this line, otherwise good to go